mlir/tools/mlir-miopen-driver/mlir-miopen-driver.cpp

```diff
@@ -510,7 +510,7 @@ static FuncOp createCPUConvolution(ModuleOp &module, OpBuilder &builder,
 
   auto charType = builder.getIntegerType(8);
   // Emit Constant ops for letters used in layouts
-  //  %g = constant 1: i8
+  //  %g = constant 103: i8
```

Comment:
It may be a better idea to amend the comment to say we are emitting the ASCII codes for those letters.